src/Backend/Modules/FormBuilder/Ajax/SaveField.php

```diff
@@ -264,7 +264,7 @@ public function execute(): void
         if ($label !== '') {
             $settings['label'] = \SpoonFilter::htmlspecialchars($label);
         }
-        if (isset($values)) {
+        if (count($values) > 0) {
```

Comment:
I see an if check for  `$values !== ''` somewhere above. Doing a count on a non-array/non-countable will fail from PHP >7.2 with `count(): Parameter must be an array or an object that implements Countable`

If we are not sure its an array or countable, we could use `if (!empty($values)) {` ?

Reply:
Good catch, I'll check if I can make it fail :p